TombEngine/Game/camera.cpp

```diff
@@ -212,6 +212,16 @@ short GetCurrentFOV()
 	return CurrentFOV;
 }
 
+void AlterRoll(short value)
```

Comment:
Not sure if we need two layers of one-line functions. Can you directly modify/get roll within script module?